sandbox/apparmor/apparmor.go

```diff
@@ -366,6 +366,12 @@ func probeKernelFeatures() ([]string, error) {
 			}
 		}
 	}
+	if data, err := os.ReadFile(filepath.Join(rootPath, featuresSysPath, "policy", "permstable32")); err == nil {
+		permstableFeatures := strings.Split(string(data), " ")
```

Comment:
```suggestion
		permstableFeatures := strings.Split(string(strings.TrimSpace(data)), " ")
```
otherwise you'll get newlines in the list of features

Reply:
`strings.Fields` was actually perfect for this, handling any kind of whitespace anywhere in the file. See: https://github.com/snapcore/snapd/pull/13821/commits/21ff1ae653deb08ea9ee2b19e83b4dd46b5a6611